shared/log4shell_scanner

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+ARCH=`arch`
+/bin/log4shell_scanner_${ARCH} $*
```

Comment:
```suggestion
#!/bin/sh
ARCH="$(arch)"
/bin/log4shell_scanner_${ARCH} "$@"
```

Reply:
`arch` is a weird one, it seems to exist in our UBI base image and is documented as part of coreutils, but it doesn't seem to be packaged at all on Arch (although `man uname` refers to it?!).

Would it make sense to use `uname --machine` instead?

Reply:
Sensible suggestions. Yes, uname --machine is likely to have wider support over different Linux distros.